### DISCUSSION TITLE
Question: Link to PowerDNS?

### DISCUSSION THREAD

**Discussion by Berndinox** [ngoduykhanh/PowerDNS-Admin#1350] [Q&A] [answered]
Can i preconfig the PowerDNS API Access via Variables inside Docker? Or Config-File/Secret?

would love to make the Container stateless

**Comment by jbe-dw** [comment, marked as answer]
Hello,

The setting is in the database. There is no way to configure it via another method. This mean that you have to bootstrap the setting via a SQL query when you provide the database for the first time.

Regards
